.github/workflows/ci.yml

```diff
@@ -23,8 +20,10 @@ jobs:
         version:
           - '1.6'
           - '1'
+          - 'nightly'
         os:
           - ubuntu-latest
+          - windows-latest
```

Comment:
I think we also need to add a test for MacOs also

Reply:
Agreed!